code/game/atoms.dm

```diff
@@ -36,7 +36,7 @@ var/global/list/ghdel_profiling = list()
 	//Detective Work, used for the duplicate data points kept in the scanners
 	var/list/original_atom
 
-	var/list/beams
+	var/list/beams = list()

```

Comment:
But seriously though how did you think it wasn't important to mention adding 500,000 lists to the game